### DISCUSSION TITLE
Is webview2 running with admin privilege?

### DISCUSSION THREAD

**Discussion by Huangpuguang** [MicrosoftEdge/WebView2Feedback#4026] [Q&A]
Hi,if my application is running with admin privileges，when starting the webview2 subprocess, does the webview2 subprocess also run with admin privileges?

**Comment by novac42** [comment]
Thanks for reaching out. I've assigned this to a dev that can help follow up on this.

**Comment by LiangTheDev** [comment]
We will try to run the webview2 sub process without admin privilege. You can test it out.
